src/libs/pulumi-cli.ts

```diff
@@ -42,8 +43,15 @@ export async function downloadCli(range: string): Promise<void> {
   core.info(`Configured range: ${range}`);
 
   const { version, downloads } = await getVersionObject(range);
+
   core.info(`Matched version: ${version}`);
 
+  const isUnsupportedVersion = semver.lt(version, '3.0.0');
```

Comment:
This looks very good. Thank you!

We should probably move `3.0.0` to a separate variable, so we can reference the same version number when we are bumping (helps with release). I can do a follow-up PR :)

Reply:
Sound reasonable 😃 Awesome, thanks!